fastly/errors.go

```diff
@@ -11,147 +11,150 @@ import (
 
 // ErrMissingService is an error that is returned when an input struct requires
 // a "Service" key, but one was not set.
-var ErrMissingService = errors.New("Missing required field 'Service'")
+var ErrMissingService = errors.New("missing required field 'Service'")
 
 // ErrMissingStatus is an error that is returned when an input struct requires
 // a "Status" key, but one was not set.
-var ErrMissingStatus = errors.New("Missing required field 'Status'")
+var ErrMissingStatus = errors.New("missing required field 'Status'")
 
 // ErrMissingTag is an error that is returned when an input struct requires
 // a "Tag" key, but one was not set.
-var ErrMissingTag = errors.New("Missing required field 'Tag'")
+var ErrMissingTag = errors.New("missing required field 'Tag'")
 
 // ErrMissingVersion is an error that is returned when an input struct requires
 // a "Version" key, but one was not set.
-var ErrMissingVersion = errors.New("Missing required field 'Version'")
+var ErrMissingVersion = errors.New("missing required field 'Version'")
 
 // ErrMissingContent is an error that is returned when an input struct requires a
 // "Content" key, but one was not set.
-var ErrMissingContent = errors.New("Missing required field 'Content'")
+var ErrMissingContent = errors.New("missing required field 'Content'")
 
 // ErrMissingLogin is an error that is returned when an input struct requires a
 // "Login" key, but one was not set.
-var ErrMissingLogin = errors.New("Missing required field 'Login'")
+var ErrMissingLogin = errors.New("missing required field 'Login'")
 
 // ErrMissingName is an error that is returned when an input struct requires a
 // "Name" key, but one was not set.
-var ErrMissingName = errors.New("Missing required field 'Name'")
+var ErrMissingName = errors.New("missing required field 'Name'")
 
 // ErrMissingKey is an error that is returned when an input struct requires a
 // "Key" key, but one was not set.
-var ErrMissingKey = errors.New("Missing required field 'Key'")
+var ErrMissingKey = errors.New("missing required field 'Key'")
```

Comment:
Do note that these are part of the exported API. I assume that it is acceptable to change the error value, right?

Reply:
I personally think this is acceptable and don't think it constitutes a breaking change as consumers _should_ be asserting against the types token and not the value.